src/components/CalendarPopover/CalendarPopover.st.css

```diff
@@ -76,17 +73,34 @@
     display: none;
 }
 
-.root:isShown {
-    display: block;
+.contentContainer {
+    box-sizing: border-box;
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-top: value(CurrentVerticalPadding);
+    padding-right: value(CurrentHorizontalPadding);
+    padding-bottom: value(CurrentVerticalPadding);
+    padding-left: value(CurrentHorizontalPadding);
 }
 
-.root:animated:isShown {
-    animation: bounce-in 0.2s ease 0s 1 normal ;
-    animation-fill-mode: forwards;
+.content {
+    margin-top: 16px;
+    overflow-y: auto;
+    flex-grow: 0;
+    flex-shrink: 1;
+    flex-basis: auto;
 }
 
-.root:withShadow {
-    box-shadow: 0 0 25px 10px rgba(80, 80, 80, 0.1), 0 4px 50px 0 rgba(209, 209, 209, 0.1);
+.title {
+    -st-mixin: TPAText(
+            MainTextFont value(CurrentTitleFont),
+            MainTextColor value(CurrentTitleColor)
+    );
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: auto;
```

Comment:
the title is HTML prop now. so no Text or style for text is needed.

Reply:
That's true 👍 
After your PR will be merged i'll update this one accordingly